filecoin-proofs/tests/api.rs

```diff
@@ -484,6 +485,82 @@ fn run_seal_pre_commit_phase1<Tree: 'static + MerkleTreeTrait>(
     Ok((piece_infos, phase1_output))
 }
 
+fn run_proof<Tree: 'static + MerkleTreeTrait>(
+    config: PoRepConfig,
```

Comment:
As a refactor just to decompose the work more, this seems fine, but the name `run_proof` is misleading, since this is also bundling unsealing, which is definitely not part of the proof.

Reply:
It would probably be better to pull the unsealing part out into its own function.